evap/evaluation/templates/base.html

```diff
@@ -185,6 +202,40 @@
             };
         </script>
 
+        <script type="module">
+            // Notebook logic
```

Comment:
Any chance this could be a file `notebook.ts` instead? I haven't looked at the implementation details too close, but for other UI elements we have adopted a pattern of having a class that contains all the logic (for example the contact modal)